### DISCUSSION TITLE
Is it possible to import an icon explicitly? (@tabler/icons-react)

### DISCUSSION THREAD

**Discussion by mrclrchtr** [tabler/tabler-icons#1244] [Q&A]
I'm using @tabler/icons-react.

I would like to import an icon directly instead of using an index import. Something like this:

```typescript
import { IconUserPlus } from "@tabler/icons-react/icons/IconUserPlus";
```

Is that somehow possible?

The reason for this is that the loading times in my development environment (vite) are too slow. I have read up on it and it is probably due to the fact that tree shaking is not possible in dev mode and therefore all icons have to be loaded when index importing. https://stackoverflow.com/a/76280863/1469497

**Comment by mrclrchtr** [comment]
ok, that seems to be this issue...  https://github.com/tabler/tabler-icons/issues/1233

**Comment by aliakseikislou** [comment]
Would love to see this implemented as well.
